src/interactivity/onecore/ConIoSrvComm.cpp

```diff
@@ -242,7 +243,7 @@ NTSTATUS ConIoSrvComm::EnsureConnection()
 VOID ConIoSrvComm::ServiceInputPipe()
 {
     // Save off a handle to the thread that is coming in here in case it gets blocked and we need to tear down.
-    THROW_HR_IF(E_NOT_VALID_STATE, _inputPipeThreadHandle); // We can't store two of them, so it's invalid if there are two.
+    THROW_HR_IF(E_NOT_VALID_STATE, !!_inputPipeThreadHandle); // We can't store two of them, so it's invalid if there are two.

```

Comment:
I trust you. But I'm confused by the `!!`. Maybe we should add a comment saying why?

Reply:
I don't know what warning it was because your branch didn't spin an official build with my change, but my guess is that the problem was the compiler warning about a `HANDLE` (which is a pointer) being used when a `bool` is expected.

`!!whatever` is a C idiom to "bool-ify" an integer or pointer value. I know what it is and use it but I understand how obscure it can be.